homeassistant/components/wiffi/wiffi.py

```diff
@@ -0,0 +1,193 @@
+"""Parser for wiffi telegrams and server for wiffi devices."""
```

Comment:
Please extract the device specific interface code and data parsing to a standalone library published on PyPI.

https://developers.home-assistant.io/docs/creating_component_code_review#4-communication-with-devicesservices

Reply:
Moved device interface code into pypi library.